src/engine-actions/get-uncommitted-pending-balance.js

```diff
@@ -7,15 +7,14 @@ const { Big } = require('../utils')
  */
 async function getUncommittedPendingBalance () {
   const { unconfirmedBalance } = await walletBalance({ client: this.client })
-  const { pendingClosingChannels = [], pendingForceClosingChannels = [], waitingCloseChannels = [] } = await listPendingChannels({ client: this.client })
+  const { pendingForceClosingChannels = [] } = await listPendingChannels({ client: this.client })
 
-  if (pendingClosingChannels.length === 0 && pendingForceClosingChannels.length === 0 && waitingCloseChannels.length === 0) {
-    this.logger.debug('getUncommittedPendingBalance: No channels exist')
+  if (!pendingForceClosingChannels.length) {
+    this.logger.debug('getUncommittedPendingBalance: No pendingForceClosingChannels exist')
     return Big(unconfirmedBalance).toString()
   }
 
-  const allPendingChannels = pendingClosingChannels.concat(pendingForceClosingChannels).concat(waitingCloseChannels)
-  const totalPendingLocalBalance = allPendingChannels.reduce((acc, c) => {
+  const totalPendingLocalBalance = pendingForceClosingChannels.reduce((acc, c) => {
```

Comment:
I would add some commentary here about the values that we are ignoring, and why (they are zero, or they are already included in `unconfirmedBalance`